### PR TITLE
Feature/storybook6 badge

### PR DESCRIPTION
**Pull request by Kevintirta**
Purpose: make badge component for storybook 6

### CODE REVIEW

File: src/General/Badge/BadgeStyle.ts [hunk 1]
```diff
@@ -5,13 +5,15 @@ import { BadgeVariant } from './BadgeVariant';
 export const BadgeContainer = styled.div<BadgeContainerProps>`
   position: relative;
   display: inline-flex;
+  align-items: center;
```
Comment (ritikamotwani) [inline src/General/Badge/BadgeStyle.ts]: @Kevintirta , why the changes in the styles?

File: src/General/Badge/Badge.tsx [hunk 2]
```diff
@@ -26,10 +26,16 @@ const Badge: React.FunctionComponent<Props> = ({
   </React.Fragment>
 );
 
-interface Props extends React.ComponentPropsWithoutRef<typeof BadgeContainer> {
+export type BadgeType = 'dimmed' | 'default';
+
+export interface Props
+  extends React.ComponentPropsWithoutRef<typeof BadgeContainer> {
+  /** Set the label of Badge. */
```
Comment (ritikamotwani) [inline src/General/Badge/Badge.tsx]: Sets the label of the Badge.

File: src/General/Badge/Badge.tsx [hunk 3]
```diff
@@ -26,10 +26,16 @@ const Badge: React.FunctionComponent<Props> = ({
...
   label: string | number;
+  /** Set Badge into superscript type. */
```
Comment (ritikamotwani) [inline src/General/Badge/Badge.tsx]: Sets badge to the superscript type.

**Comment by ritikamotwani** [comment]
@Kevintirta the superscript type looks much different due to the style changes http://aries.glints.design/?path=/story/general--badge. Any reason for changing the styles? 🤔 

Rest are minor reviews.
Thanks!

**Comment by Kevintirta** [comment]
> @Kevintirta the superscript type looks much different due to the style changes http://aries.glints.design/?path=/story/general--badge. Any reason for changing the styles? 🤔
> 
> Rest are minor reviews.
> Thanks!

Not sure why but with currently styling (before any changes), the content of the badge look not align vertically, is this the expected behaviour. As shown in the below picture
Screenshot 2020-12-09 at 8 04 55 PM

File: src/General/Badge/BadgeStyle.ts (hunk 1 above)
Comment (Kevintirta) [reply src/General/Badge/BadgeStyle.ts]: seems that the content of the label does not align vertically but, the new changes also break the superscript type. I have screenshotted the component in the comment section below

**Comment by ritikamotwani** [comment]
Hi @Kevintirta !
I see your point. The content is very much shifted to the top. But with these changes the superscript tag becomes of more height. 
Let the scope of this PR be just to upgrade the documentation and not change the styles, and open an issue for this here https://github.com/glints-dev/glints-aries/issues/541 with a `bug` label.

**Comment by ritikamotwani** [comment]
By the way, the documentation changes look good, thanks!

**Comment by Kevintirta** [comment]
> Hi @Kevintirta !
> I see your point. The content is very much shifted to the top. But with these changes the superscript tag becomes of more height.
> Let the scope of this PR be just to upgrade the documentation and not change the styles, and open an issue for this here https://github.com/glints-dev/glints-aries/issues with a `bug` label.

Okay understood, I have removed the styling change and have made another issue for the UI (https://github.com/glints-dev/glints-aries/issues/541), Thank you Ritika!

**Review by ritikamotwani — APPROVED** [review]
